src/table/hooks/use-model-store.ts

```diff
@@ -0,0 +1,17 @@
+const keyStore = (initialState: { [key: string]: EngineAPI.IField } = {}) => {
```

Comment:
I don't think this is a hook as it does not have to obey the rules of hooks

Reply:
Indeed